packages/paste-website/src/components/tokens-list/TokensExample.tsx

```diff
@@ -17,10 +17,33 @@ export const ColorBox: React.FC<BackgroundColor> = ({backgroundColor}) => {
   return <Absolute backgroundColor={backgroundColor} padding="space50" preset="fill" />;
 };
 
-type BorderBoxProps = Pick<BoxProps, 'borderColor' | 'borderWidth'>;
-export const BorderBox: React.FC<BorderBoxProps> = ({borderColor, borderWidth}) => {
+type BorderBoxProp = Pick<BoxProps, 'borderColor' | 'borderWidth'>;
+interface BorderBoxProps extends BorderBoxProp {
+  boxColor: string;
+}
+export const BorderBox: React.FC<BorderBoxProps> = ({boxColor, borderColor, borderWidth}) => {
+  let isInverse = false;
+  if (boxColor.includes('inverse')) {
+    isInverse = true;
+  }
   return (
-    <Box borderStyle="solid" borderColor={borderColor} borderWidth={borderWidth || 'borderWidth20'} padding="space60" />
+    <Absolute
+      backgroundColor={isInverse ? 'colorBackgroundBodyInverse' : 'colorBackgroundBody'}
+      padding="space60"
+      preset="fill"
+      css={{
```

Comment:
You aughta be able to pass these styles as props since Absolute is just Box

Reply:
@TheSisb easy change. Taking care of the `css` props in this file.